examples/tutorials/notebook/notebook_tutorial_2D_crm.ipynb

```diff
@@ -1566,7 +1542,9 @@
     }
    ],
    "source": [
-    "x_slicing_force_distribution.plot(x=\"X\", y=\"totalCumulative_CD_Curve\", xlabel=\"X\", ylabel=\"CD\", figsize=(10, 5))"
+    "x_slicing_force_distribution.plot(\n",
```

Comment:
why are these changes in this PR? What are they for?

Reply:
The black formatter for jupyter notebook is included and these notebook changes are caused by the formatting.